requirements.txt

```diff
@@ -1,6 +1,33 @@
-matplotlib==3.0.2
-PySide==1.2.4
-# PySide2==5.12.0
-Shiboken==1.2.2
```

Comment:
Is this correct? I'm fairly certain we use shiboken, but I don't know if it needs to be listed here.

Reply:
From what I can see it's used only in cpp files, already handled by cmake